[EDU-5527] improve link command guide (#1332)
aziontech/docs: src/content/docs/en/pages/devtools/cli/azion-cli/commands/link-project/link.mdx

```

:::note 

> If you want to deploy your application later, run 'azion deploy'. Your application will not be available on the edge until it's deployed.

Reviewer: ```suggestion
If you want to deploy your application later, run 'azion deploy'. Your application won't be available on the edge until it's deployed.
```
